### ISSUE TITLE
Städte-Support

### ISSUE THREAD

**Issue by andneuma** [kjoscha/lichtenberg#24]
hey,

ich fänd es richtig cool, wenn man städte-support hätte. konkret: auswahl einer stadt über die startseite (pop-up), städteliste wird autogeneriert aus lat/lon, also reverse-geocoded. in python hab ich sowas mal mit osm-daten gebastelt, dabei hab ich einem query-server die lat/lon-daten geschickt, der hat mir dazu in xml die geodaten gesendet, die hab ich dann mittels parser ausgelesen. dabei hab ich dann ein dictionary erstellt, welches ich einfach nach der gewünschten raumeinheit (stadt, bundesland, land, etc. pp) abfragen konnte. war natürlich alles einigermaßen komfortabel, da es die ganzen pakete bereits gibt.
geht sicherlich auch einfacher (und hässlicher) mit regexp, wie das unter js funktioniert weißt nur da. aber php hat sicherlich auch packages für so xml-parsing-geschichten, oder?

**Comment by kjoscha** [comment]
ja, das wäre cool. kann ich mit dem integrierten geocoder irgendwie basteln...

**Comment by andneuma** [comment]
Yeah!
